helix-core/src/main/java/org/apache/helix/manager/zk/ZKUtil.java

```diff
@@ -75,7 +80,7 @@ public static boolean isClusterSetup(String clusterName, HelixZkClient zkClient)
       logger.info("Fail to check cluster setup : zookeeper client is null!");
       return false;
     }
-    ArrayList<String> requiredPaths = new ArrayList<String>();
+    ArrayList<String> requiredPaths = new ArrayList<>();
```

Comment:
Nit, shall we declare type `List<>` ?

Reply:
It's fine - the idea is not changing the existing implementation. But this is a quick fix. I'll change to List.